src/utils/fetch.ts

```diff
@@ -1,4 +1,6 @@
-// @ts-ignore
+// @ts-nocheck don't check fetch
+// TODO remove this file when fetch is supported in Node 18
```

Comment:
Let's just remove it in this version.

We can leave this to the users if they are on <18.

Reply:
I'll leave it in this PR and do some refactors in the next PR (#18)